### PR TITLE
Reserve tag 4 in vulnerability.proto

### PR DESCRIPTION
**Pull request by noahklein**

### CODE REVIEW

File: proto/v1/vulnerability.proto [hunk 1]
```diff
@@ -94,6 +94,7 @@ message Vulnerability {
   }
 
   // ID 4 is unusable due to a proto conflict.
+  reserved 4;
```
Comment (tylerdatgoogle) [inline proto/v1/vulnerability.proto]: Please move this to the top of the message for easier visibility. We eventually would want all reserved proto fields in the same place.

File: proto/v1beta1/vulnerability.proto [hunk 2]
```diff
@@ -91,6 +91,7 @@ message Vulnerability {
   }
 
   // ID 4 is unusable due to a proto conflict.
+  reserved 4;
```
Comment (tylerdatgoogle) [inline proto/v1beta1/vulnerability.proto]: same comment

File: proto/v1/vulnerability.proto [hunk 3]
```diff
@@ -45,6 +45,8 @@ enum Severity {
 
 // Vulnerability provides metadata about a security vulnerability in a Note.
 message Vulnerability {
+  reserved 4;
```
Comment (aysylu) [inline proto/v1/vulnerability.proto]: Please comment this out, but keep the wording.

Comment (tylerdatgoogle) [reply proto/v1/vulnerability.proto]: Are you saying to comment out the line "reserved 4"? The motive for this PR is to add this tag using the reserved keyword: https://developers.google.com/protocol-buffers/docs/proto#reserved.

Comment (aysylu) [reply proto/v1/vulnerability.proto]: @tylerdatgoogle and I chatted about this and reached an agreement that we should revert this PR. Sorry for the troubles, @noahklein. My concern here is balancing good OSS stewardship and ensuring compatibility with internal Grafeas implementations. Making this an un-commented `reserved` field would conflict with internal usage and implications for other Grafeas users. Commenting out the `// reserved 4` has the same effect as the state before, so it's a no-op. I filed an issue for myself to look into this more comprehensively. Thank you both!

**Pull request closed without merging**